src/Uno.UI/UI/Xaml/FrameworkElement.Interface.wasm.cs

```diff
@@ -112,18 +111,8 @@ public Brush Background
 		}
 
 		protected virtual void OnBackgroundChanged(DependencyPropertyChangedEventArgs e)
-			// Warning some controls (eg. CalendarViewBaseItem) takes ownership of the background rendering.
-			// They override the OnBackgroundChanged and explicitly do not invokes that base method.
-			=> SetAndObserveBackgroundBrush(e.NewValue as Brush);
-
-		private protected void SetAndObserveBackgroundBrush(Brush brush)
 		{
-			var subscription = _backgroundSubscription ??= new SerialDisposable();
-
-			subscription.Disposable = null;
-			subscription.Disposable = BorderLayerRenderer.SetAndObserveBackgroundBrush(this, brush);
 		}
```

Comment:
It's unclear to me why this method could still be useful.

Reply:
The virtual method is overriden in up the chain